index.html

```diff
@@ -195,7 +201,7 @@ <h3 id="about-header">About me</h3>
   </div>
  
     <!-- stack item -->
-    <section>
+    <div id="stack-cards">
```

Comment:
I would recommend decreasing the size of the cards. Currently, they are too big, and scaling on the large screen makes them even worse. Kindly fix them according to the Figma template of your choosing.
This is how it is looking now.
![image](https://user-images.githubusercontent.com/33268927/173057487-553bc1f5-a1b5-4d58-8be8-00bd6f5b6896.png)